src/banknames.json

```diff
@@ -1,188 +1,1294 @@
   "BCEY": "Bank of Ceylon",
-  "BDBL": "Bandhan Bank Limited",
+  "BCUB": "Bicholim Urban Co-operative Bank",
+  "BCUX": "Baghat Urban Co-operative Bank",
+  "BDBL": "Bandhan Bank",
+  "BDBX": "Bellary District Coop Central Bank",
+  "BDCX": "Bijapur District Central Co-operative Bank",
+  "BDDX": "Bhadradri Co-operative Urban Bank",
+  "BDIX": "Bhandara District Central Co-operative Bank",
+  "BDNX": "Bhind Nagrik Sahakari Bank Mydt",
+  "BDOX": "Banda District Co-operative Bank Limited Banda",
+  "BDUX": "Banda Urban Co Operative Bank",
+  "BEDX": "Beed District Central Co-operative Bank",
+  "BELX": "Belgaum District Central Co-operative Bank",
+  "BFUX": "Bhagyodaya Friends Urban Coop Bank",
+  "BGBX": "Bihar Gramin Bank",
+  "BGCX": "Balageria Central Coop Bank",
+  "BGGX": "Baroda Gujarat Gramin Bank",
+  "BGUX": "Begusarai Central Co-operative Bank",
+  "BGVX": "Bangiya Gramin Vikash Bank",
+  "BHAX": "Bhagyodaya Co Operative Bank",
+  "BHBX": "Bhilai Nagarik Sahakari Bank Maryadit",
+  "BHCX": "Bhuj Commercial Co-operative Bank",
+  "BHDX": "Bhadohi Urban Co Operative Bank Gyanpur",
+  "BHEX": "Bhel Employees Co-operative Bank",
+  "BHGX": "Bhagalpur Central Co-operative Bank",
+  "BHIX": "Bhilwara Mahila Urban Co Operative Bank",
+  "BHJX": "Bhuj Mercentile Coop Bank",
+  "BHMX": "Brahmadeodada Mane Sahakari Bank Solapur",
+  "BHOX": "Bhopal Co Operative Central Bank",
+  "BHRX": "Bharatpur Central Co-operative Bank",
+  "BHSX": "Bharati Sahakari Bank Pune",
+  "BHTX": "Bhatkal Urban Co-operative Bank",
+  "BHUX": "Bhilwara Urban Co-operative Bank",
+  "BHWX": "Bhiwani Central Co-operative Bank Bhiwani",
+  "BJUX": "Bijnor Urban Co Operative Bank",
+  "BKCX": "Bhavasara Kshatriya Co-operative Bank",
   "BKDN": "Dena Bank",
+  "BKDX": "Banaskantha Dist Central Coop Bank",
   "BKID": "Bank of India",
-  "BMBL": "Bharatiya Mahila Bank Ltd",
+  "BKSX": "Baran Kendriya Sahakari Bank Baran",
+  "BLGX": "Shree Basaveshwar Co-operative Bank",
+  "BMBL": "Bharatiya Mahila Bank",
+  "BMCB": "Bombay Mercantile Co-operative Bank",
+  "BMCX": "Barmer Central Co-operative Bank",
+  "BMPX": "Banaskantha Mercantile Co-operative Bank",
+  "BMSX": "Bhagyalakshmi Mahila Sahakari Bank",
+  "BNBX": "Betul Nagrik Sahakari Bank Mydt",
+  "BNCX": "Bhatpara Naihati Co Operative Bank",
   "BNPA": "BNP Paribas Bank",
+  "BNSX": "Bhagini Nivedita Sahakari Bank",
+  "BODX": "Bagalkot District Central Co-operative Bank",
   "BOFA": "Bank of America",
-  "BOTM": "Bank of Tokyo Mitsubishi Ltd",
+  "BORX": "Boral Union Co Operative Bank",
+  "BOTM": "Bank of Tokyo Mitsubishi",
+  "BOTX": "Shree Botad Mercantile Coop Bank",
+  "BPCX": "Bhadgaon People's Co-operative Bank",
+  "BPSX": "Bilagi Pattana Sahakari Bank Niyamit",
+  "BRCX": "Bhavana Rishi Coop. Urban Bank",
+  "BRDX": "Baroda Central Co-operative Bank",
+  "BRGX": "Baroda Rajasthan Kshetriya Gramin Bank",
+  "BRMX": "Bramhapuri Urban Co Operative Bank",
+  "BRUX": "Bharuch District Central Coop Bank Bharuc",
```

Comment:
@sunnyagg Another reason:

- `508812` IIN corresponds to `KKBK` in our systems currently (RuPay)
- The NPCI excel marks it as: `AGRX, AGROHA COOPERATIVE URBAN BANK LTD	CO-OP, KKBK0ACUB01, 508812`) (`Code, Name, IFSC, IIN`)
- We can switch over the Bank code in our iins to `AGRX` and everything should become better easily.